.drone.star

```diff
@@ -96,6 +116,15 @@ def installPnpm():
         ],
     }]
 
+def lint():
+    return [{
+        "name": "lint",
```

Comment:
`name` varible here and in L95 inside function `pnpmlint()` is same. Is it a good practice to have same name. Maybe change one of them so that there wont be any confusion.

Reply:
I have changed. Can you check if I have done correctly or not ?